src/services/lnd/utils.ts

```diff
@@ -115,8 +115,19 @@ export const lndBalances = async (lnd: AuthenticatedLnd) => {
     sumBy(channel.close_payments, (payment) => (payment.is_pending ? payment.tokens : 0)),
   )
 
+  // adds 330 sats for every selfInitiatedChannel even if not all channels use anchor
+  const { channels } = await getChannels({ lnd })
+  const selfInitiatedChannels = channels.filter(
+    ({ is_partner_initiated }) => is_partner_initiated === false,
+  )
+  let anchorBalance = 0
+  for (let selfInitiatedChannel of selfInitiatedChannels) {
+    anchorBalance += 330
+  }
```

Comment:
Const satsAnchorOutput = 330
Let anchorBalance = selfInitiatedChannel * satsAnchorOutput

Reply:
I have used a loop because selfInitiatedChannels is an array.

Reply:
Ok then you can just take the length of the array

Reply:
thanks for the suggestion, made the change in [be1f13c](https://github.com/GaloyMoney/galoy/pull/1520/commits/be1f13c8748a814e458375c2bde6117898cf140b) and tests are passing locally